api/http/handler/stacks/stack_update.go

```diff
@@ -125,7 +130,7 @@ func (handler *Handler) stackUpdate(w http.ResponseWriter, r *http.Request) *htt
 
 	err = handler.DataStore.Stack().UpdateStack(stack.ID, stack)
```

Comment:
we may want to update a few props before updating a stack, here is what we have in EE:
```
user, err := handler.DataStore.User().User(securityContext.UserID)
if err != nil {
return &httperror.HandlerError{StatusCode: http.StatusBadRequest, Message: "Cannot find context user", Err: errors.Wrap(err, "failed to fetch the user")}
}
stack.UpdatedBy = user.Username
stack.UpdateDate = time.Now().Unix()
stack.Status = portainer.StackStatusActive
```